src/Vimeo/Vimeo.php

```diff
@@ -347,7 +347,11 @@ public function replace($video_uri, $file_path, $upgrade_to_1080 = false, $machi
     private function perform_upload($file_path, $ticket)
     {
         if ($ticket['status'] != 201) {
-            throw new VimeoUploadException('Unable to get an upload ticket.');
+            if(isset($ticket['body']['error'])){
+		$ticket_error = "[" . $ticket['body']['error'] . "]";
+	    }
+
+ 	    throw new VimeoUploadException('Unable to get an upload ticket.' . $ticket_error);
```

Comment:
Yes, I tried that case. Initially I had it initialized to an empty string but removed it when I tried it without initializing it and noticed that 'Null' or 'undefined' wasn't appended to the string.

I'll reinstate it for readability's sake...